_includes/content/v3notice.html

```diff
@@ -1,7 +1,6 @@
 <div class="v3notice">
-LoopBack 3 has entered Maintenance long-term support (LTS), only critical bugs
-and critical security fixes will be provided. We urge all LoopBack 3 users
-to migrate their applications to LoopBack 4 as soon as possible. Learn more in
-<a href="https://strongloop.com/strongblog/lb3-entered-maintenance-mode/">
-our blog post</a>.
+LoopBack 3 has reached End-of-Life. No new features, bug fixes, or security
+fixes will be provided or accepted. We urge all LoopBack 3 users to migrate
+their applications to LoopBack 4 as soon as possible. Learn more about
```

Comment:
IBM will keep providing LB3 support for customers using IBM API Connect product. Let's use the following message please:

```suggestion
LoopBack 3 has reached end of life. We are no longer accepting pull requests or providing 
support for community users. The only exception is fixes for critical  bugs and security 
vulnerabilities provided as part of support for IBM API Connect customers.
We urge all LoopBack 3 users to migrate their applications to LoopBack 4 as soon as possible. 
Learn more about
```